ports/esp8266/esp_mphal.h

```diff
@@ -101,6 +101,12 @@ void mp_hal_pin_open_drain(mp_hal_pin_obj_t pin);
 } while (0)
 #define mp_hal_pin_read(p) pin_get(p)
 #define mp_hal_pin_write(p, v) pin_set((p), (v))
-
+static inline int mp_hal_pin_output_value(mp_hal_pin_obj_t pin) {
```

Comment:
As above, I suggest calling this `mp_hal_pin_read_output`.

Reply:
Done